nixos/modules/services/networking/charybdis.nix

```diff
@@ -51,46 +51,55 @@ in
         '';
       };
 
+      motd = mkOption {
+        type = types.nullOr types.lines;
+        default = null;
+        description = "Charybdis MOTD.";
```

Comment:
I've expanded the description to include the path to the MOTD file we're using.
This isn't really useful for use in ircd.conf, since charybdis doesn't allow MOTD location to be configured there, but is still good to include for users who'd rather write the file themselves rather than letting nix manage it.